src/core/src/core_logic/ExecutionConfig.py

```diff
@@ -179,3 +182,27 @@ def __check_and_create_temp_folder_if_not_exists(self):
             self.composite_logger.log_debug("Temp folder does not exist, creating one from extension core. [Path={0}]".format(str(self.temp_folder)))
             os.mkdir(self.temp_folder)
 
+    def __is_package_eula_accepted(self):
+        """ Reads customer provided config on EULA acceptance from disk and returns a boolean.
+            NOTE: This is a temporary solution and will be deprecated no later than TBD date"""
+        if not os.path.exists(Constants.AzGPSPaths.EULA_SETTINGS):
+            self.composite_logger.log_debug("NOT accepting EULA for any patch as no corresponding EULA Settings found on the VM")
+            return False
+
+        try:
+            eula_settings = json.loads(self.env_layer.file_system.read_with_retry(Constants.AzGPSPaths.EULA_SETTINGS) or 'null')
+            self.composite_logger.log_debug("EULA config values read from disk: AcceptEULAForAllPatches=[{0}] AcceptedBy=[{1}] LastModified=[{2}]"
+                                            .format(str(Constants.EulaSettings.ACCEPT_EULA_FOR_ALL_PATCHES), str(Constants.EulaSettings.ACCEPTED_BY),
+                                                    str(Constants.EulaSettings.LAST_MODIFIED)))
+            if eula_settings is not None \
+                    and Constants.EulaSettings.ACCEPT_EULA_FOR_ALL_PATCHES in eula_settings \
+                    and eula_settings[Constants.EulaSettings.ACCEPT_EULA_FOR_ALL_PATCHES] is True:
+                self.composite_logger.log_debug("Accept EULA set to True in customer config")
+                return True
+            else:
+                self.composite_logger.log_debug("Accept EULA not found to be set to True in customer config")
+                return False
```

Comment:
Given how carefully we're rationalizing logs, this can be a single log_debug.

4 lines of code:
- read (same as here)
- evaluate return to a variable 
- log and include the true/false as one param
- return variable

Message can be (reminder that the bracket struct is [<key>=<value>]... vs. <key>=[<value>] here): 
"EULA config values from disk: [AcceptEULAForAllPatches={0}] [AcceptedBy={1}] LastModified=[{2}]"  -- with actual values from disk.

Reply:
Done